src/components/DayPicker.jsx

```diff
@@ -436,38 +631,63 @@ export default class DayPicker extends React.Component {
     const transformType = this.isVertical() ? 'translateY' : 'translateX';
     const transformValue = `${transformType}(${translationValue}px)`;
 
+    const shouldFocusDate = !isCalendarMonthGridAnimating && !showKeyboardShortcuts;
+
     return (
-      <div className={dayPickerClassNames} style={dayPickerStyle} >
+      <div className={dayPickerClassNames} style={dayPickerStyle}>
         <OutsideClickHandler onOutsideClick={onOutsideClick}>
-          {!verticalScrollable && this.renderNavigation()}
-
-          <div className="DayPicker__week-headers">
+          <div
```

Comment:
Can we put a pin on it until we get design to look at it (slash do it in a follow up PR)? I feel like this is not critical here.